atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -332,7 +335,7 @@ class ReactorConfig(object):
     DEFAULT_CONFIG = {ReactorConfigKeys.VERSION_KEY: 1}
 
     def __init__(self, config=None):
-        self.conf = config or self.DEFAULT_CONFIG
+        self.conf = deepcopy(config) or self.DEFAULT_CONFIG
```

Comment:
Should `deepcopy` include default config too? `self.conf = deepcopy(config or self.DEFAULT_CONFIG)`

Reply:
I'm not sure it really matters, but sure.

Reply:
It does! Because if you create two instances of `ReactorConfig` without providing a `config`, they'll both point to the same object since `self.DEFAULT_CONFIG` is a class attribute.

Reply:
Fixed.